api/tests/routes/backoffice/finance_test.py

```diff
@@ -1058,8 +1058,15 @@ class GetOverpaymentIncidentTest(GetEndpointHelper):
         "venue_factory,expected_xpf_text",
         [(offerers_factories.VenueFactory, ""), (offerers_factories.CaledonianVenueFactory, "(1205 CFP)")],
     )
-    def test_get_incident(self, authenticated_client, venue_factory, expected_xpf_text):
-        finance_incident = finance_factories.FinanceIncidentFactory(venue=venue_factory())
+    @pytest.mark.parametrize(
+        "zendesk_id",
+        [
+            None,
+            "1",
+        ],
+    )
```

Comment:
Détail : il y a moyen de prendre moins de place :)
```suggestion
    @pytest.mark.parametrize("zendesk_id", [None, "1"])
```